GoogleForCreators/web-stories-wp
# CTA (Outlink) improvements

## Summary
- Adds `theme` setting
- Add `icon` setting
- Replaces using `amp-story-page-attachment` with `amp-story-page-outlink`
- Ensure protocol usage for the link


## Relevant Technical Choices
- Currently uses just a checkbox with the option to use dark theme (default being light theme) -- not sure if this is the best control to use for it.
- The link icon is not automatically pulling in the icon, it needs to be set manually / otherwise displays default. Should we change this?


## To-do
- [ ] Confirm UX



## User-facing changes
![Screenshot 2021-08-04 at 12 25 07](https://user-images.githubusercontent.com/3294597/128157195-677b6d4c-866d-44c1-8093-3a2e8a67b466.png)
![Screenshot 2021-08-04 at 12 25 02](https://user-images.githubusercontent.com/3294597/128157201-7e7de2cf-2c06-4efc-a63b-92d23931091d.png)
![Screenshot 2021-08-04 at 12 24 40](https://user-images.githubusercontent.com/3294597/128157205-0e8a163b-7a99-40a0-83e9-8acfe2c3f2e0.png)
![Screenshot 2021-08-04 at 12 30 09](https://user-images.githubusercontent.com/3294597/128157836-e93da28b-d177-42ed-846e-2b0f19b0e724.png)




## Testing Instructions
1. Before checking out this PR, add a Page Attachment and modify some settings.
2. Now check out this PR and verify the previous is still working as expected.
3. Use the new fields -- the dark theme checkbox and the link icon.
4. Verify both appear correctly in the editor and in the preview, too.
5. Verify that if you add a link without protocol, it is added automatically when blurring.


### QA






This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #8120 
Fixes #8377
